recipe/meta.yaml

```diff
@@ -116,36 +71,24 @@ requirements:
     - m2-patch                           # [win]
     - jom                                # [win]
     - curl                               # [win]
-    - perl 5.*
+    - perl
     # Add readline here, so that newest readline is used.
     # Otherwise build and host might end up with two different versions
     # If they do, merge_build_host will fail.
     - readline                           # [linux]
-    # libclang is required in order to enable simultaneous x86_64 and
-    # arm64 support for QDoc
-    - libclang                           # [arm64]
-    # Although these should be host-only dependencies, they are needed when
-    # cross-compiling arm64 binaries on x86_64 since llvm-config (host/arm64)
-    # cannot be executed on x86_64 to locate libclang successfully
-    - llvmdev                           # [arm64]
-    - clangdev                          # [arm64]
+    - ninja
   host:
     # For pkg-config gl, fontconfig + freetype
     - pthread-stubs                      # [linux]
     - dbus                               # [linux]
     - fontconfig                         # [linux]
     - freetype                           # [linux]
-    - glib
-    - gst-plugins-base
-    - gstreamer
-    - pulseaudio                         # [linux]
+    - libglib                            # [linux]
```

Comment:
Somebody once told me that typically, one would require glib at build time, and that would export the requirement of `libglib`.